packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -88,8 +93,9 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
   hydrationContext: HydrationContext<C, CX>,
   scheduleWork: (fiber: Fiber, expirationTime: ExpirationTime) => void,
   computeExpirationForFiber: (fiber: Fiber) => ExpirationTime,
+  actualRenderTimer: ActualRenderTimer | null,
```

Comment:
Let's always pass it the module and assume that the compiler will strip it all out.